haskell/src/Monocle/Api/Server.hs

```diff
@@ -27,12 +27,30 @@ import Monocle.Search.Syntax (ParseError (..))
 import qualified Monocle.UserGroup as UserGroupPB
 import Proto3.Suite (Enumerated (..))
 
+-- | /api/2/about endpoint
+configGetAbout :: ConfigPB.GetAboutRequest -> AppM ConfigPB.GetAboutResponse
+configGetAbout = const response
+  where
+    response = do
+      config <- getConfig
+      -- TODO(fbo) - fetch Monocle version from cabal file
```

Comment:
here is how i did it for podenv: https://github.com/podenv/podenv/pull/68/files# . And since we don't build the final result with nix, we can drop the `th-env` part.

Reply:
Thanks I'll handle it in a follow-up commit